test/Dialect/SPIRV/canonicalize.mlir

```diff
@@ -126,6 +126,19 @@ func @extract_array_interm() -> (vector<2xi32>) {
 
 // -----
 
+// CHECK-LABEL: extract_from_not_constant
+func @extract_from_not_constant() -> i32 {
+  // CHECK: %[[VAR:.*]] = spv.Variable
+  // CHECK-NEXT: %[[COMPOSITE:.*]] = spv.Load "Function" %[[VAR]]
+  // CHECK-NEXT spv.CompositeExtract %[[COMPOSITE]]
```

Comment:
Missing `:` after `CHECK-NEXT`. Actually I think we just need to check the `spv.CompositeExtract` op exists, so no need to check `spv.Variable` and `spv.Load`. :)

Reply:
@antiagainst thanks for review, fixed!